sub/source/docs/pilot-control.rst

```diff
@@ -0,0 +1,55 @@
+.. _pilot-control:
+
+=============
+Pilot Control
+=============
+
+Control of the vehicle can be accomplished in three ways:
+
+- Via Ground Control Stations using:
+   - GCS attached Joysticks, or command lines, to send MAVLink `RC override <https://mavlink.io/en/messages/common.html#RC_CHANNELS_OVERRIDE>`__ commands that act as RC inputs (even if no RC receiver is attached)
+   - MAVLink ` MANUAL CONTROL <https://mavlink.io/en/messages/common.html#MANUAL_CONTROL>`__ messages which includes axis commands and button states.
+
+- RC control (not enabled by default)
+
+Pilot Control Inputs
+====================
+If RC is not enabled, pilot inputs are via the GCS joystick, controlling the vehicle axes, and body-frame movements laterally,forward/back, and up/down in depth. Additionally, lights and camera can be controlled from the GCS.
+
+If RC is enabled, then the pitch/roll/yaw/throttle/lateral/forward inputs are sent from the RC transmitter and mapped from RC channels to these inputs via the ``RCMAP_`` parameters.
+
+Pilot Control Freedoms by Mode
+==============================
+The following assumes the axis is controllable. Some frame configurations do not provide all axes.
```

Comment:
```suggestion
The following assumes each axis is controllable. Some frame configurations do not provide all axes (e.g. a vehicle with only horizontal thrusters may not be able to pitch or roll).
```
Likely worth including an example/explanation around what not providing all axes means / why a frame may not do so. There are already plenty of support questions around using position-required modes for vehicles without a positioning sensor - this seems in a similar vein 😅